src/platformAccessory.ts

```diff
@@ -84,11 +96,11 @@ export class RinnaiControlrPlatformAccessory {
             .updateValue(this.targetTemperature);
 
         this.service.getCharacteristic(this.platform.Characteristic.CurrentTemperature)
-            .onGet(this.getTargetTemperature.bind(this))
-            .updateValue(this.targetTemperature)
+            .onGet(this.getOutletTemperature.bind(this))
+            .updateValue(this.outletTemperature)
             .setProps({
-                minValue: this.minValue,
-                maxValue: this.maxValue,
+                minValue: THERMOSTAT_MIN_CURRENT_TEMP,
+                maxValue: THERMOSTAT_MAX_CURRENT_TEMP,
```

Comment:
Was it intentional to use constants vs the user defined values?

Reply:
Yes, the current temp could be out of the range for target temperature. For example, when water heater has been idling for a while, the current temperature (outlet temperature) will be close to the main water line temperature, which is much lower than the minimal target temperature.